### PR TITLE
Added search for portfolio tests

### PR DESCRIPTION
**Pull request by yury-dubinin**
## What is the purpose of the change:

Due to portfolio page change added a search for a token for 2 reasons:
- make sure search works
- make sure token gets to a view

### CODE REVIEW

File: packages/web/e2e/pages/portfolio-page.ts [hunk 1]
```diff
@@ -61,4 +64,10 @@ export class PortfolioPage extends BasePage {
       await this.page.waitForTimeout(1000);
     }
   }
+
+  async searchForToken(tokenName: string) {
+    await this.searchInput.fill(tokenName);
+    // we expect that after 2 seconds tokens are loaded and any failure after this point should be considered a bug.
+    await this.page.waitForTimeout(2000);
```
Comment (p0mvn) [inline packages/web/e2e/pages/portfolio-page.ts:71]: Would there be a clear feedback about why the test failed? If not, is there anywhere we could add a comment or a log?

File: packages/web/e2e/tests/portfolio.wallet.spec.ts [hunk 2]
```diff
@@ -58,33 +58,46 @@ test.describe("Test Portfolio feature", () => {
   });
 
   test("User should be able to see native balances", async () => {
+    await portfolioPage.searchForToken("OSMO");
```
Comment (p0mvn) [inline packages/web/e2e/tests/portfolio.wallet.spec.ts:61]: Nit: we could probably put these in a loop to minimize cognitive overhead stemming from code duplication

**Review by p0mvn — APPROVED** [review]
LGTM - thank you

File: packages/web/e2e/tests/portfolio.wallet.spec.ts (hunk 2 above)
Comment (yury-dubinin) [reply packages/web/e2e/tests/portfolio.wallet.spec.ts]: agree, I just need to do it. Moved to parameteried tests approach.

**Pull request merged**